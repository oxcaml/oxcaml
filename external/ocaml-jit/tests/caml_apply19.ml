let force_caml_apply19 _p0 _p1 _p2 _p3 _p4 _p5 _p6 _p7 _p8 _p9 _p10 _p11
      _p12 _p13 _p14 _p15 _p16 _p17 _p18 _p19 = ()

let x : _ -> unit =
  (Sys.opaque_identity force_caml_apply19)
    0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18
;;
