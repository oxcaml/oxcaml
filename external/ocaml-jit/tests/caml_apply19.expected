val force_caml_apply19 :
  'a ->
  'b ->
  'c ->
  'd ->
  'e ->
  'f ->
  'g ->
  'h ->
  'i ->
  'j -> 'k -> 'l -> 'm -> 'n -> 'o -> 'p -> 'q -> 'r -> 's -> 't -> unit =
  <fun>
val x : '_weak1 -> unit = <fun>
